functions/Get-DbaAgentJob.ps1

```diff
@@ -4,11 +4,12 @@ FUNCTION Get-DbaAgentJob {
 			Gets SQL Agent Job information for each instance(s) of SQL Server.
 
 		.DESCRIPTION
-			The Get-DbaAgentJob returns connected SMO object for SQL Agent Job information for each instance(s) of SQL Server.
+			The Get-DbaAgentJob returns connected SMO object for SQL Agent Job information for each instance(s) of SQL Server. The default behavior
+			is to return enabled jobs only.
```

Comment:
A function should do what is says, in this case, get the Agent Jobs, and not implement an arbitrary filter by default just because it is convenient for some people.